libexec/tfenv-version-name

```diff
@@ -137,4 +137,8 @@ if [[ ! -d "${TFENV_CONFIG_DIR}/versions/${TFENV_VERSION}" ]]; then
   log 'debug' "version '${TFENV_VERSION}' is not installed (set by ${TFENV_VERSION_SOURCE})";
 fi;
 
+if [[ "${TFENV_VERSION}" = min-required ]]; then
```

Comment:
Did you mean:
`if [[ "${TFENV_VERSION}" == "min-required" ]]; then` ?

Reply:
It's the same right? No problem if you prefer though.